index.bs

```diff
@@ -581,23 +592,23 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
 
-1. Let |currentlyAvailableAuthenticators| be a new [=ordered set=] consisting of all [=authenticators=] 
+1. Let |currentlyAvailableAuthenticators| be a new [=ordered set=] consisting of all [=authenticators=]
```

Comment:
mumble.  from an editing perspective, to me, folding lines at some agreed-to length, and re-formatting them as we go along such that the prose parags are easier to read trumps pretty much everything else.  this is part of why I do kdiff3 text-only diffs and also use daisydiff and do not rely on only github's source diff display....